### PR TITLE
fix(shrinkwrap): remove shrinkwrap file

### LINKED ISSUE #277: Dependencies are not hoisted
I've noticed that the dependencies of decktape are never hoisted. 

For example:

```
mkdir tmp && cd tmp
npm i --save-exact decktape@3.7.0
tree node_modules/ -L 3
node_modules/
└── decktape
    ├── node_modules
    │   ├── puppeteer
    │   ├── puppeteer-core
    [...]
    │   └── ylru
    ├── npm-shrinkwrap.json
    ├── package.json
    └── plugins
```

This is a problem when you later install a version of puppeteer yourself, it gets duplicated in your node_modules:

```bash
npm i --save-exact puppeteer-core@19.7.3
tree node_modules/ -L 3
node_modules/
└── decktape
    ├── node_modules
    │   ├── puppeteer # 👈 here
    │   ├── puppeteer-core
    [...]
    ├── package.json
    └── plugins
├── puppeteer # 👈 and here
├── puppeteer-core  
[...]
```

... which is a problem, because installing 2 versions of puppeteer simultaneously doesn't seem to work:

```
npm WARN cleanup Failed to remove some directories [
npm WARN cleanup   [
npm WARN cleanup     'D:\\a\\1\\s\\node_modules\\decktape\\node_modules\\puppeteer',
npm WARN cleanup     [Error: EBUSY: resource busy or locked, rmdir 'D:\a\1\s\node_modules\decktape\node_modules\puppeteer'] {
npm WARN cleanup       errno: -4082,
npm WARN cleanup       code: 'EBUSY',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'D:\\a\\1\\s\\node_modules\\decktape\\node_modules\\puppeteer'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'D:\\a\\1\\s\\node_modules\\decktape\\node_modules',
npm WARN cleanup     [Error: EBUSY: resource busy or locked, rmdir 'D:\a\1\s\node_modules\decktape\node_modules\puppeteer'] {
npm WARN cleanup       errno: -4082,
npm WARN cleanup       code: 'EBUSY',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'D:\\a\\1\\s\\node_modules\\decktape\\node_modules\\puppeteer'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'D:\\a\\1\\s\\node_modules\\decktape',
npm WARN cleanup     [Error: EBUSY: resource busy or locked, rmdir 'D:\a\1\s\node_modules\decktape\node_modules\puppeteer'] {
npm WARN cleanup       errno: -4082,
npm WARN cleanup       code: 'EBUSY',
npm WARN cleanup       syscall: 'rmdir',
npm WARN cleanup       path: 'D:\\a\\1\\s\\node_modules\\decktape\\node_modules\\puppeteer'
npm WARN cleanup     }
npm WARN cleanup   ],
npm WARN cleanup   [
npm WARN cleanup     'D:\\a\\1\\s\\node_modules',
npm WARN cleanup     [Error: EBUSY: resource busy or locked, rmdir 'D:\a\1\s\node_modules\decktape\node_modules\puppeteer'] {
npm WARN cleanup   ]
npm WARN cleanup ]
npm ERR! code 1
npm ERR! path D:\a\1\s\node_modules\puppeteer
```

I'm reasonably sure this hoisting behavior is caused by the [npm-shrinkwrap.json](https://github.com/astefanutti/decktape/blob/master/npm-shrinkwrap.json) file. Is this file there by choice? Or can we change it to a `package-lock.json` file?

### LINKED ISSUE #341: Bug: `npm ci` doesn't work with decktape@3.13.0
Strange problem. It seems that `npm ci` doesn't work with the latest decktape release on linux and windows. 

decktape

This is probably a bug npm in combination with the [npm-shrinkwrap.json](https://github.com/astefanutti/decktape/blob/master/npm-shrinkwrap.json) file. 

This is my thinking
- You've used mac os to release decktape
- `npm-shrinkwrap.json` includes `fsevents`, because it is included on mac (and only mac).
- `npm install decktape` will install decktape ignoring the `fsevents`, because I'm installing on linux.
- `npm ci` failed because of a bug.

I'll create an issue over at npm.

In the meantime, is there some way to solve this problem?

### PR DESCRIPTION
**Pull request by nicojs**
The shrinkwrap file is no longer needed... ?

Closes #277
Closes #341

### CODE REVIEW

**Comment by astefanutti** [comment]
Could you remove the reference to the shrinkwrap file from the Dockefile please?

**Comment by nicojs** [comment]
Ah yes. I did. I've also updated the node version in docker (to 20.15).

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 WORKDIR /decktape
 
-COPY package.json npm-shrinkwrap.json ./
+COPY package.json package-lock.json ./
 COPY libs libs/
 COPY plugins plugins/
 COPY decktape.js ./
 
-RUN npm install
+RUN npm ci --omit=dev
```
Comment (nicojs) [inline Dockerfile:15]: By switching to `ci --omit=dev` we achieve 2 things:

1. The docker pipeline fails when the lock-file is out of sync with the package.json file.
2. We omit playwright and other `devDependencies` from installing

**Review by astefanutti — APPROVED** [review]

**Comment by nicojs** [comment]
A friendly reminder to please create a release when you have the time @astefanutti 😘

**Comment by astefanutti** [comment]
@nicojs apologies for the delay. I've just released v3.14.0.

**Pull request merged**